druid/src/test/java/org/apache/calcite/test/DruidAdapterIT.java

```diff
@@ -3058,15 +3058,15 @@ private void testCountWithApproxDistinct(boolean approx, String sql, String expe
         + "(count(distinct \"user_id\") * 2) from \"wiki\"";
     wikiApprox(sql)
         .queryContains(druidChecker(druid))
-        .returnsUnordered("EXPR$0=-10590");
+        .returnsUnordered("EXPR$0=100");
```

Comment:
Is this change of result expected?

Reply:
will look into it and verify it.

Reply:
fixed, the cause of failure was missing hyperUnique column in the latest druid default wikipedia ingestion spec, have added the column by copying the ingestion spec file in test vm.